dspace/config/spring/api/subscriptions_email_configuration.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+           http://www.springframework.org/schema/util
+           http://www.springframework.org/schema/util/spring-util.xsd">
+
+    <bean class="org.dspace.subscriptions.SubscriptionEmailNotificationServiceImpl">
+        <constructor-arg name="subscriptionType2generators">
+            <map>
+                <entry key="content" value-ref="contentNotifyGenerator"/>
+            </map>
+        </constructor-arg>
+        <constructor-arg name="contentUpdates">
+            <map>
+                <entry key="community" value-ref="communityUpdates"/>
+                <entry key="collection" value-ref="collectionUpdates"/>
+            </map>
+        </constructor-arg>
+    </bean>
+
+    <bean id="contentNotifyGenerator" class="org.dspace.subscriptions.ContentGenerator" />
+
+    <util:map id="contentUpdates">
+        <entry key="community" value-ref="communityUpdates"/>
+        <entry key="collection" value-ref="collectionUpdates"/>
+    </util:map>
+
+    <util:list id="generalMetadataEntity">
+        <value>dc.title</value>
+    </util:list>
+
+    <util:map id="entityDissemination">
```

Comment:
Here's where the main bug is.  This list only supports Entity Types, and doesn't work for a default Item.

Reply:
This bug has been addressed, generic Item is handled as fallback